index.js

```diff
@@ -16,6 +16,16 @@
  */
 function Fireproof(firebaseRef, promise) {
 
+  if (!Fireproof.Promise) {
+
+    try {
+      Fireproof.Promise = Promise;
```

Comment:
will do.

On Mon, Nov 23, 2015 at 3:18 PM Josh Dague <notifications@github.com> wrote:

> In index.js
> <https://github.com/casetext/fireproof/pull/16#discussion_r45676118>:
>
> > @@ -16,6 +16,16 @@
> >   */
> >  function Fireproof(firebaseRef, promise) {
> >
> > +  if (!Fireproof.Promise) {
> > +
> > +    try {
> > +      Fireproof.Promise = Promise;
>
> I think just checking that it's a function would suffice. Just a simple
> sanity check in case someone accidentally Promise = true;
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/casetext/fireproof/pull/16/files#r45676118>.
>